app/components/ui/switch.tsx

```diff
@@ -2,15 +2,11 @@
 
 import * as SwitchPrimitives from "@radix-ui/react-switch";
 import { cn } from "app/lib/utils/cn";
-import {
-  forwardRef,
-  type ComponentPropsWithoutRef,
-  type ElementRef,
-} from "react";
+import * as React from "react";
```

Comment:
Revert these boss, import only what you need from React

Reply:
Don't import all of React

Reply:
Don't import all of React

Reply:
Though this isn't my component sir

Reply:
Should I make the changes?